### PR TITLE
Move to GitHub workflow

### PR DESCRIPTION
**Pull request by timtroendle**
As neither Travis nor Goodtables.io work reliably, I am trying a GitHub workflow here.

### CODE REVIEW

**Comment by timtroendle** [comment]
There is currently an error in the datapackage (even if small). That's a convincing sign that we _should_ have automatic checks!

**Comment by timtroendle** [comment]
@richard-thonig This seems to work. I'll keep this as a mandatory check, if fine with you.

File: csp-guru/datapackage.json [hunk 1]
```diff
@@ -460,7 +460,7 @@
     ],
     "licenses": [
         {
-            "name": "CC-BY 4.0",
+            "name": "CC-BY-4.0",
```
Comment (richard-thonig) [inline csp-guru/datapackage.json:463]: I'm not convinced scheinbar kann man auch gar keine bindestiche machen.
Aber passt schon.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/repolicy/csp-guru.svg?branch=master)](https://travis-ci.org/repolicy/csp-guru)
+![Datapackage](https://github.com/repolicy/csp-guru/actions/workflows/frictionless.yaml/badge.svg)
```
Comment (richard-thonig) [inline README.md:1]: sounds good.

**Review by richard-thonig — APPROVED** [review]
thx!

**Pull request merged**